### PR TITLE
Fix bower repo names, settings generation and prevent cache

### PR DESCRIPTION
**Pull request by kaizo**

### CODE REVIEW

File: grunt_tasks/hashAndReplaceFiles.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+var hashAndReplaceFiles = function (grunt, files, whereToReplace) {
```
Comment (jordins) [inline grunt_tasks/hashAndReplaceFiles.js]: Duplicated file from desktop, we should extract it into an npm library.

File: src/js/main.js [hunk 2]
```diff
@@ -46,36 +45,34 @@ require([
...
+		translator.applyTranslations();
+		var login = new Login(),
+			recover = new Recover(),
 			redirector = new Redirector(),
 			credentials = new Credentials(),
 			prepare = new Prepare();
-			prepare.showLoading();
+
+		prepare.showLoading();
+		var params = prepare.getUrlParametersByNames(['username', 'token']);
```
Comment (jordins) [inline src/js/main.js]: please, get the url path (/password/reset)

File: src/js/prepare.js [hunk 3]
```diff
@@ -241,20 +226,51 @@ define([
...
+		}
+	};
+
 	Prepare.prototype.prepareRegisterButton = function () {
 		if (this.platformSettings.enableUserRegistration) {
 			$('#register_button').removeClass('hidden');
 		}
 	};
 
+	Prepare.prototype.getUrlParametersByNames = function(names) {
```
Comment (jordins) [inline src/js/prepare.js]: maybe use getURLParameter (redirector file)?

File: src/js/translator.js [hunk 4]
```diff
@@ -50,32 +53,42 @@ define(["js/tr", "i18next"], function (tr) {
...
-		$("#detectingEyerun").text(tr($('#detectingEyerun').text().trim()));
+		$("#textLogIn").text(tr($("#textLogIn").text().trim()));
 		$("#captchaRefreshButton").attr('alt', tr($("#captchaRefreshButton").attr('alt')));
 
-		$("#eyeRunDown").html(tr($("#eyeRunDown").html()));
-		$("#eyeRunLinkor").html(tr($("#eyeRunLinkor").html()));
+	};
+
+	Translator.prototype.messageTranslation = function(message) {
+		return tr(message);
```
Comment (jordins) [inline src/js/translator.js:94]: why this wrapper?

File: src/js/main.js [hunk 5]
```diff
@@ -80,10 +80,12 @@ require([
 		}
 
 		prepare.showLoading();
-		var params = prepare.getUrlParametersByNames(['username', 'token']);
+		if(location.pathname === "/password/reset/") {
```
Comment (jordins) [inline src/js/main.js]: Can this path be in settings?

Comment (absis) [inline src/js/main.js]: Done

File: src/js/main.js (hunk 2 above)
Comment (absis) [inline src/js/main.js]: Done

File: src/js/prepare.js (hunk 3 above)
Comment (absis) [inline src/js/prepare.js]: Done

**Pull request merged**